README.org

```diff
@@ -377,17 +377,22 @@ feature that does not need synchronization, you can skip this step.
 
 **** WebDAV
 
-organice has support for WebDAV and ships with a Docker container with
-a WebDAV server based on Apache. You can make use of that and use this
-WebDAV back-end for local development.
+organice has support for WebDAV and you can make use of that for local development.
+We offer instructions for two options for setting up WebDAV servers but you
+may use any other standard solution. 
+
+- Option 1: A Docker Container with Apache
```

Comment:
These two lines are collapsed to one. Can you make two sentences out of it or just use:

1. A Docker container w Apache: …
2. A WebDAV server in go...